plugins/ruby-foreman-docker/debian/changelog

```diff
@@ -1,3 +1,9 @@
+ruby-foreman-docker (3.2.0-1) stable; urgency=low
+
+  * 3.2.0 released
+
+ -- Sebastian Graessl <mail@bastilian.me> Tue Jun 27 16:17:47 2017
```

Comment:
there needs to be one more space before the date, also the date format is wrong.